specification/subscription/resource-manager/readme.md

```diff
@@ -26,17 +26,17 @@ These are the global settings for the Subscription API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2017-11-preview
+tag: package-2018-03-01-preview
 ```
 
 
-### Tag: package-2017-11-preview
+### Tag: package-2018-03-01-preview
```

Comment:
We shouldn't replace the existing tags, we need to add new tags for our new release.